.ruby-version

```diff
@@ -0,0 +1 @@
+3.1.2
```

Comment:
Is this necessary? People should be free to use whatever they please

Reply:
No this is not necessary this get commited by mistake. I'll drop it, in the next commit.